### PR TITLE
Fix salt and streamline setup process

### PR DESCRIPTION
**Pull request by CameronRP**
- Now uses script in salt `state-apply-test.sh` to configure image
- Streamline process for making image

### CODE REVIEW

File: image-configure.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+echo "Installing saltstack"
+scp install-saltstack.sh $userHost:
+ssh $userHost "sudo ./install-saltstack.sh"
+ssh $userHost "sudo rm ./install-saltstack.sh"
+
+echo "Applying salt state"
+wget https://github.com/TheCacophonyProject/saltops/archive/prod.zip
+unzip prod.zip
+cd saltops-prod/
+sed -i '/maybe-reboot/d' salt/top.sls # We don't want to reboot when making the image
```
Comment (mjs) [inline image-configure.sh:19]: How does top.sls get there if you're only downloading the prod branch? top.sls is only in master right?

**Review by mjs — COMMENTED** [review]
Looks awesome - just one question.

File: image-configure.sh (hunk 1 above)
Comment (CameronRP) [reply image-configure.sh:19]: We maintain two `top.sls` files now. One in the Master branch and one in the other branches. This was done when `state-apply-test.sh` was added.

https://github.com/TheCacophonyProject/saltops/blob/prod/salt/top.sls

Comment (mjs) [reply image-configure.sh:19]: ah yes... I forgot about that

**Review by mjs — APPROVED** [review]

**Pull request merged**